apache-httpd/vhost.conf

```diff
@@ -121,5 +124,16 @@ ServerName localhost
         <Location /osgp-adapter-ws-smartmetering/wsdl>
             SSLVerifyClient none
         </Location>
+
+        # OSGP Secret Management
+        <Location /osgp-secret-management>
+            SSLVerifyClient require
+            SSLVerifyDepth 1
+            SSLOptions +ExportCertData
+        </Location>
+        <Location /osgp-secret-management/ws/SecretManagement/secretManagement.wsdl>
```

Comment:
Adapt to /osgp-secret-management/wsdl

Reply:
I deployed some of the components in a local environment.

Using the following URL I am able to access the WSDL:
https://localhost/osgp-secret-management/ws/SecretManagement/secretManagement.wsdl

I can't find a working URL under:
https://localhost/osgp-secret-management/wsdl

For smart metering management, for instance, the following gives a WSDL:
https://localhost/osgp-adapter-ws-smartmetering/wsdl/SmartMeteringManagement.wsdl

If you can show a URL for the adapted path that you propose, where the WSDL can be retrieved, I think I'll update it.
For now I assume the difference is needed, because the osgp-secret-management web services are set up another way than the others.

Reply:
In the WebServiceConfig class the URI is defined as `/ws/SecretManagement` so the location used here seems correct to me.